languages/python/concepts/lists/about.md

```diff
@@ -112,21 +112,36 @@ Items inside lists (_like the sequence types `string` and `tuple`_), can be acce
 
 >>> breakfast_foods = ["Oatmeal", "Fruit Salad", "Eggs", "Toast"]
 
-#Oatmeal is at index 0 or index -4
+# Oatmeal is at index 0 or index -4.
 >>> first_breakfast_food = breakfast_foods[0]
 'Oatmeal'
 
 >>> first_breakfast_food = breakfast_foods[-4]
 'Oatmeal'
 ```
 
+You can access a portion of the list with the slice notation `[start:stop]`. The slice of a list from start to stop is defined as the sequence of items with index `i` such that `start <= i < stop`.
+You can also get a slice of a list from `start` to `stop` with step `step` with the notation `[start:stop:step]`.
+A slice of a list is a copy of a portion of the original list so you do not modify the original but you get a new list with the elements you asked for.
+
+```python
+
+>>> colors = ["Red", "Purple", "Green", "Yellow", "Orange", "Pink", "Blue", "Grey"]
+
+>>> middle_colors = colors[2:6]
```

Comment:
```suggestion
# If there is no step parameter, the step is assumed to be 1
>>> middle_colors = colors[2:6]
```

Reply:
Thanks you for thé suggestions ! Should I also apply them to the introduction file?

Reply:
ooh!  Yes, please.  I haven't (yet) read that far!  😄